src/osd/OSD.cc

```diff
@@ -1660,6 +1660,13 @@ bool OSD::asok_command(string command, cmdmap_t& cmdmap, string format,
     } else {
       op_tracker.dump_ops_in_flight(f);
     }
+  } else if (command == "dump_blocked_ops") {
+    if (!op_tracker.tracking_enabled) {
+      ss << "op_tracker tracking is not enabled";
```

Comment:
@gregsfortytwo . I see this PR(https://github.com/ceph/ceph/pull/2440) about optimize optracker and it already merged into master. I think  we should remove dynamic control optracer(suppose now optracker don't  have move overhead and suppose print blocked ops is more important than the overhead of optracker).  What's your advise?